backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/VdsManager.java

```diff
@@ -1293,11 +1323,25 @@ public void updateV2VJobInfos(List<V2VJobInfo> v2vJobInfos) {
         }
     }
 
+
     private void initAvailableCpus() {
         List<VdsCpuUnit> cpuTopology = cachedVds.getCpuTopology();
         if (cpuTopology == null) {
             return;
         }
         this.cpuTopology = cpuTopology;
+        this.cpuTopology.stream().filter(cpu -> cpu.getCpu() == Integer.parseInt(cachedVds.getVdsmCpusAffinity()))
+                .forEach(cpu -> cpu.pinVm(Guid.SYSTEM, CpuPinningPolicy.MANUAL));
```

Comment:
please extract `Integer.parseInt(cachedVds.getVdsmCpusAffinity())` as it doesn't change within the iteration and it's less readable that way, in a follow up patch